docs/installing-nightly.md

```diff
@@ -2,6 +2,8 @@
 
 >**Note**: only install the nightly if you'd like to try the bleeding edge capabilities of bicep. These are much more likely to have undiscovered bugs or other issues. If you find anything, please open an issue.
 
+>**Note**: The Bicep VS code extension versions older than 0.2 must be uninstalled before or after the installation of the new version.
```

Comment:
Want to add something like - "otherwise you will see duplicate and/or inconsistent errors being reported"

Reply:
Yeah, makes sense.